src/app/containers/EpisodeList/index.jsx

```diff
@@ -34,35 +34,29 @@ const StyledEpisodeListItem = styled.li`
   }
 `;
 
-const EpisodeList = ({
-  children,
-  script,
-  service,
-  dir,
-  darkMode,
-  ulProps,
-  liProps,
-}) => {
-  if (!children.length) return null;
+const EpisodeList = forwardRef(
+  ({ children, script, service, dir, darkMode, ulProps, liProps }, ref) => {
+    if (!children.length) return null;
 
-  const hasMultipleChildren = children.length > 1;
+    const hasMultipleChildren = children.length > 1;
 
-  return (
-    <EpisodeContext.Provider value={{ script, service, dir, darkMode }}>
-      {hasMultipleChildren ? (
-        <StyledEpisodeList role="list" {...ulProps}>
-          {children.map(child => (
-            <StyledEpisodeListItem key={child.key} {...liProps}>
-              {child}
-            </StyledEpisodeListItem>
-          ))}
-        </StyledEpisodeList>
-      ) : (
-        children
-      )}
-    </EpisodeContext.Provider>
-  );
-};
+    return (
+      <EpisodeContext.Provider value={{ script, service, dir, darkMode }}>
+        {hasMultipleChildren ? (
+          <StyledEpisodeList role="list" ref={ref} {...ulProps}>
```

Comment:
I think the ref has to be forwarded here since `EpisodeList` is a functional component, but I completely missed the conditional! I think what I'll do is pass the ref to the `Spacer` which wraps the whole component, and remove the `forwardRef` since the `Spacer` is just a styled `<aside>` 👍